README.md

```diff
@@ -7,14 +7,19 @@
 
 
 ## Inspiration 🧑‍🎨
+Developers are always looking for solutions to make our lives easier, but sometimes it becomes strenous for them to continue focusing on the building solutions. A small relaxation between their work could be a hgreat help in reducing the monotonous stress as well as give them  a good laugh.
 
 ## What it does ⛳
+Meme O Meme is a website solely designed for the developers who are stuck in a pike of work and get no or very less time to release their stress. This website directs them towards a short series of memes of their preference and tries to give them the relaxation they need.
 
 ## How we built it 🚧
+We used ReactJS, Tailwindcss, Figma to built this website. 
```

Comment:
`We used ReactJS, Tailwindcss, Figma to built this website. ` => `We used ReactJS, Firebase, Figma to build this website. `